jenkins/infra/jenkins/build-ci-image-all.groovy

```diff
@@ -43,46 +43,60 @@ pipeline {
 
 		stage('build ci images') {
 			parallel {
-				stage('cpp') {
+				stage('cpp - ubuntu:22.04') {
 					steps {
 						script {
-							dispatchBuildCiImageJob('cpp')
+							dispatchBuildCiImageJob('cpp', 'ubuntu:22.04')
```

Comment:
can you not use default argument instead of specifying 22.04 everywhere?

Reply:
Yea, I had a default parameter in first commit but then change it to make it more readable.
Since the stage names are updated, it make sense to use the default parameter.